UPGRADE.md

```diff
@@ -117,7 +117,7 @@ in your product feed plugin because the instances of `StandardFeedItemInterface`
 
 ### [shopsys/coding-standards]
 #### From 3.x to 4.0
-- In order to run all checks, there is new unified way - execute `php vendor/bin/ecs check /path/to/project --config=vendor/shopsys/coding-standards/easy-coding-standard.neon`
+- In order to run all checks, there is new unified way - execute `php vendor/bin/ecs check /path/to/project --config=vendor/shopsys/coding-standards/easy-coding-standard.yml`

```

Comment:
coding-standards version 4.0 was already released. You cannot change upgrade log of already released tag ;-)
So please revert these changes and create new section #### From [7.0.0-alpha2 to Unreleased]